newIDE/app/src/EventsFunctionsExtensionEditor/EventsFunctionConfigurationEditor/EventsFunctionParametersEditor.js

```diff
@@ -39,6 +39,8 @@ type Props = {|
   onParametersUpdated: () => void,
   helpPagePath?: string,
   freezeParameters?: boolean,
+  makeMoveFreeEventsParameter?: boolean,
+  makeMoveBehaviorEventsParameter?: boolean,
```

Comment:
Seems like these are never used? 🤔

Also, don't pass makeXXX function, pass instead the created functions:
`onMoveFreeEventsParameter={makeMoveFreeEventsParameter(i18n)}`

A `makeXXX` function mean "I'm a function that you should call with my argument (18n) so that I send you another function that you can use (here, pass as a prop, so that the children don't care about i18n)".